pkgs/top-level/python-packages.nix

```diff
@@ -610,6 +634,16 @@ in {
     };
   };
 
+  aiohttp_1_0 = self.aiohttp.override rec {
```

Comment:
Is this version really needed? Having multiple versions of a library starts to become problematic as soon as they would end up in one closure.